scripts/hourly_wmb.py

```diff
@@ -71,4 +76,15 @@
                         help="Turn on diagnostic mode (no commits)")
 
     args = parser.parse_args()
-    run(args)
+    log = setup_logging(args.log_level, args.log, args.error_email)
+
+    engine = create_engine(args.connection_string)
+    Session = sessionmaker(engine)
+    sesh = Session()
+    sesh.begin_nested()
+
+    # Pipeline
+    for file in download(args):
+        for row in normalize(file):
+            for obs in align(sesh, row):
```

Comment:
This isn't quite right. The way you have expressed this there are multiple files per download, and there multiple observations *per row* from the align phase. There should be only *one* obs per row. See [my pipeline PR](https://github.com/pacificclimate/crmprtd/pull/28) for how I think the pipeline should be.